src/cfg/settings.py

```diff
@@ -0,0 +1,64 @@
+import os
+from urllib.parse import unquote
+
+import requests
+from dotenv import load_dotenv
+
+if os.path.isfile("./src/cfg/env/.env"):
+    load_dotenv("./src/cfg/env/.env")
+
+security = {
+    "token": os.environ["SECURITY_TOKEN"],
+}
+
+ahgs_api = {
+    "request_headers": {
+        "Accept-Language": "en-US",
+        "User-Agent": "Diveharder API - api.diverharder.com",
+    },
+    "auth_headers": {
+        "Accept-Language": "en-US",
+        "User-Agent": "Diveharder API - api.diverharder.com",
```

Comment:
not sure you want to be advertising this 😅

Reply:
AHGS Devs can come at me - I want them to know who I am. Hopefully they see that I Get some good use outta it, they'll work with me and we can get better api access